### PR TITLE
i18n(es): Update `astro-components`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `astro-components` based on this commit c9ee2b277c8ac68556542b877bfb6bf413443ca9

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/core-concepts/astro-components.mdx [hunk 1]
```diff
@@ -245,7 +245,39 @@ import Wrapper from '../components/Wrapper.astro';
...
+
+```astro title="src/components/StockTable.astro" {5-7, 9-13} '<Fragment slot="header">' '<Fragment slot="body">'
+---
+import CustomTable from './CustomTable.astro';
+---
+<CustomTable>
+  <Fragment slot="header"> <!-- agrega un encabezado para la tabla -->
+    <tr><th>Nombre del producto</th><th>Unidades disponibles</th></tr>
+  </Fragment>
+  <Fragment slot="body"> <!-- pass table body -->
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-components.mdx:274]:
```suggestion
  <Fragment slot="body"> <!-- pasar el cuerpo de la tabla -->
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Pull request merged**